src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,10 +1,9 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String bulldozerType;
+    private int weight;
+
```

Comment:
Optional, but it'll be a good solution to write a constructor in the Machine class and inherit it in the child classes like Bulldozer, Truck etc, using the keyword `super` in the new constructor